File: salary.py
import mysql.connector
import datetime

d=datetime.datetime.now()
m=d.strftime("%B")
yr=d.strftime("%G")

db=mysql.connector.connect(host="localhost",user="gowthamidb",passwd="Chinnu@1",database="ems")
mycursor=db.cursor()

mycursor.execute("select emp.empid,emp.pay,leaves.lops from emp,leaves where emp.ename=leaves.ename order by emp.empid")

rs=mycursor.fetchall()

for i in rs:
	x=i[0]
	y=i[1]
	z=i[2]
	#print(x," ",y," ",z)
	mycursor.execute("""insert into salary(empid,basic,lops,month,year) values(%s,%s,%s,%s,%s)""",(x,y,z,m,yr))

mycursor.execute("update salary set da=basic*0.7,hra=basic*0.15,pf=basic*0.12,allow=basic*.05,ded=(lops*basic)/30,gross=basic+da+hra+allow,netsalary=gross-(ded+pf)")

db.commit()
db.close()

File: salary.php

<?php

//$date=$_POST["dt"];
$date='2021-07-06';
$yr=date('Y',strtotime($date));
$mt=date('F',strtotime($date));

?>
<center>
<h3><u>Employee Details </u></h3>
<table border=1 width="70%" align="center">
<tr>

<th>EMPID</th>
<th>Basic Pay</th>
<th>Take Home</th>
<th>DA</th>
<th>HRA</th>
<th>PF</th>
<th>Allowance</th>
<th>Deduction</th>
<th>Gross</th>
<th></th>
<th>Desired Month :<?php echo $mt; ?></th>
</tr>

<?php
echo "<body style='background-color:lightgreen'>";
$con=mysqli_connect("localhost","gowthamidb","Chinnu@1","ems");
if($con->connect_error)
{
die("Connection Failed:".$con->connect_error);
}
$sql= "SELECT * from salary where month='$mt'&& year='$yr'";
$result=$con->query($sql);

if($result->num_rows>0)
{
while($row=$result->fetch_assoc())
{
	echo"<tr><td>".$row["empid"]."</td><td>".$row["basic"]."</td><td>".$row["netsalary"]."</td><td>".$row["da"]."</td><td>".$row["hra"]."</td><td>".$row["pf"]."</td><td>".$row["allow"]."</td><td>".$row["ded"]."</td><td>".$row["gross"]."</td></tr>";
}
}
else{
echo "0 result";
}
$conn->close();
?>
</table>
